autoload/goimpl.vim

```diff
@@ -65,7 +105,8 @@ endfunction
 let g:goimpl#os_arch = get(g:, 'goimpl#os_arch', s:os_arch())
 
 function! goimpl#impl(recv, iface)
-    if !executable(g:goimpl#cmd)
+    let binpath = s:check_bin_path(g:goimpl#cmd)
+    if empty(binpath)
```

Comment:
`binpath` should also be used on command execution. [here](https://github.com/rhysd/vim-go-impl/blob/4fb6a50a4d3b3bb7112845bdae543df30225e1d8/autoload/goimpl.vim#L73)